modules/caddyhttp/push/handler.go

```diff
@@ -122,11 +122,8 @@ func (h Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhtt
 	}
 
 	// serve only after pushing!
-	if err := next.ServeHTTP(lp, r); err != nil {
-		return err
-	}
-
-	return nil
+	err := next.ServeHTTP(lp, r)
+	return err
```

Comment:
Kinda surprised no linter is saying this should simply be `return next.ServeHTTP(lp, r)`

:man_shrugging: 

Not a big deal but just something I noticed.

Reply:
there may be one that'll do that for us.  I'll have a peek :)